core/src/services/gcs/core.rs

```diff
@@ -209,6 +210,7 @@ impl GcsCore {
         path: &str,
         size: Option<u64>,
         content_type: Option<&str>,
+        cache_control: Option<&str>,
```

Comment:
How about passing `OpWrite` here to make it easier to extand?

Reply:
i not sure what is OpWrite, `mut Option<&str>` ?

Reply:
Maybe you could take a look at here, https://github.com/fatelei/opendal/blob/0d6747d428627543a2ce632542aab5a278534611/core/src/services/gcs/backend.rs#L419

It's the args of some specific operation.